### PR TITLE
Cleanup mapbox

### PR DESCRIPTION
**Pull request by mariusaarsnes**
Rewritten to be more readable by splitting functionality into multiple methods.

@MsevenDev is it good enough? **Note:** some of the changes were made in another PR so look at the whole file (`Mapbox.jsx`).

### CODE REVIEW

File: webapp/src/components/Mapbox/Mapbox.jsx [hunk 1]
```diff
@@ -47,28 +49,40 @@ const getMaxDatasetValue = dataset => {
...
   const id = tractNumber ? 'grunnkretsnummer' : 'DELOMRNR';
   geoData.features.forEach(feature => {
     const dataPointer = feature.properties[id];
     feature.properties.data = countingData[dataPointer];
   });
+
+  return geoData;
+};
+
+const InsertMobilityDataIntoMapLayer = (dataset, tractNumber, mapRef) => {
```
Comment (MsevenDev) [inline webapp/src/components/Mapbox/Mapbox.jsx]: camelCasing

Comment (MsevenDev) [reply webapp/src/components/Mapbox/Mapbox.jsx]: Ellers ser det bra ut :)

Comment (mariusaarsnes) [inline webapp/src/components/Mapbox/Mapbox.jsx]:
```suggestion
const insertMobilityDataIntoMapLayer = (dataset, tractNumber, mapRef) => {
```

File: webapp/src/components/Mapbox/Mapbox.jsx [hunk 2]
```diff
@@ -193,93 +276,41 @@ const Mapbox = ({ tractNumber, unitNumber, handleSelect, timestamp, dataset, hov
...
   useEffect(() => {
     if (!mapIsLoaded) return;
-
-    const map = mapRef.current.getMap();
-    const source = tractNumber ? 'basic-statistical-units' : 'statistical-tracts';
-
-    const geoData = InsertMobilityDataIntoMapLayerData(dataset, tractNumber);
-
-    map.getSource(source).setData(geoData);
+    InsertMobilityDataIntoMapLayer(dataset, tractNumber, mapRef);
```
Comment (mariusaarsnes) [inline webapp/src/components/Mapbox/Mapbox.jsx]:
```suggestion
    insertMobilityDataIntoMapLayer(dataset, tractNumber, mapRef);
```

**Pull request merged**